api/endpoints/login.go

```diff
@@ -15,22 +16,19 @@ func Login(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	user := new(models.User) // request user
 	if err := json.NewDecoder(r.Body).Decode(&user); err != nil {
-		w.WriteHeader(400)
-		json.NewEncoder(w).Encode(&map[string]string{"Error": err.Error()})
+		status.RespondStatus(w, 400, err)
 		return
 	}
 	var User models.User // user in database
```

Comment:
Is there a specific reason why this is named `User`? It's capitalized and is similar to `user`, doesn't seem ideal to me. Would be better off renaming this to something easier to read, unless there's a special reason why we named it `User` of course

Reply:
Fixed already